oleesir/Banking-app
# #164726709Admin should be able to View list of all bank accounts.

#### What does this PR do?
 implement accounts-template
#### Description of Task to be completed?
 implement accounts-template
#### How should this be manually tested?
* Navigate to the UI folder and click on bank_accounts.html in your browser
* You will see bank_accounts-template
#### What are the relevant pivotal tracker stories?
[Finishes #164726709](https://www.pivotaltracker.com/story/show/164726709)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="Screenshot 2019-03-25 at 8 40 37 PM" src="https://user-images.githubusercontent.com/17679511/54949003-4d99c300-4f3e-11e9-9a50-8f3238ae7778.png">
